modules/ROOT/pages/_partials/sgw-svr-compatibility.adoc

```diff
@@ -181,6 +181,15 @@ Compatibility Matrix
 
 |===
 
+[WARNING]
+====
+Starting from CBS 7.0, the `use_views` feature is no longer available.
+
+* SGW 3.1 will *still* run with `use_views` using a `default scope/default collection` configuration
```

Comment:
Not sure this makes sense. Above, you just said the `use_views` feature is no longer available.
That's not what the matrices on https://docs.couchbase.com/sync-gateway/current/compatibility.html suggest though.
Can you double check if this is just a caveat, and reword?

Reply:
it's only available - within 3.1 if you don't define S&C and only use the default S&C setup.